### PR TITLE
Issue #101: As a Devops, I would like to have a unified set of rules for yaml linting 

### PR DESCRIPTION
**Pull request by SonOfLope**
To provide conformity, we are adding a config yaml linting file that provides expected standards of yaml files for the organisation. Repositories using the reusable workflow for linting yaml files can simply omit to specify a config file path to use default config of the organisation.

Also fixes  https://github.com/ai-cfia/howard/issues/90

### CODE REVIEW

File: .github/workflows/workflow-yaml-check.yml [hunk 1]
```diff
@@ -34,7 +34,9 @@ jobs:
               if [ -n "${{ github.event.inputs.config-file-path }}" ]; then
                 yamllint -c "${{ github.workspace }}/${{ github.event.inputs.config-file-path }}" "$file"
               else
-                yamllint "$file"
+                default_config_url="https://raw.githubusercontent.com/ai-cfia/github-workflows/main/.yamllint.yml"
```
Comment (rngadam) [inline .github/workflows/workflow-yaml-check.yml:37]: https://github.com/orgs/community/discussions/63863

> To reference the script.js file, you can use the uses keyword. The uses keyword allows you to import a workflow file from another repository. In your case, you would use the following syntax:

**Review by rngadam — CHANGES_REQUESTED** [review]
using curl to download file is not necessary; use ```uses``` keyword instead to ensure that the .yamllint.yml file is present.

File: .github/workflows/workflow-yaml-check.yml [hunk 2]
```diff
@@ -25,6 +25,13 @@ jobs:
       - name: Install yamllint
         run: pip install yamllint
 
+      - name: Checkout code
+        uses: actions/checkout@v4
```
Comment (rngadam) [inline .github/workflows/workflow-yaml-check.yml:29]: checking out the entire directory seems excessive when we can just point to the .yamllint.yml file?

**Review by rngadam — CHANGES_REQUESTED** [review]

File: .github/workflows/workflow-yaml-check.yml (hunk 2 above)
Comment (rngadam) [reply .github/workflows/workflow-yaml-check.yml:29]: conversations should be resolved by the reviewer.

please refer to: 

https://github.com/orgs/community/discussions/63863

Yes, there is special handling for how myworkflow.yaml should reference script.js. When you call a reusable workflow, GitHub Actions will copy the contents of the workflow file to the current working directory. This means that the script.js file will not be available in the current working directory unless you explicitly reference it.

To reference the script.js file, you can use the uses keyword. The uses keyword allows you to import a workflow file from another repository. In your case, you would use the following syntax:

```
uses: calleerepo/.github/workflows/script.js@main
```

Comment (SonOfLope) [reply .github/workflows/workflow-yaml-check.yml:29]: I will wait until we meet and you explain your comprehension of the reference you've given. I don`t think we are trying to achieve the same thing.

Comment (rngadam) [reply .github/workflows/workflow-yaml-check.yml:30]: Yes, after reviewing this I see that we are not able to refer to the file from the called repository github-workflows directory - or at least I couldn't find a way.

**Pull request merged**